WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductCoordinator.swift

```diff
@@ -114,7 +114,9 @@ final class AddProductCoordinator: Coordinator {
     func start() {
         switch source {
         case .productsTab:
-            analytics.track(event: .ProductsOnboarding.productListAddProductButtonTapped(templateEligible: isTemplateOptionsEligible))
+                analytics.track(event: .ProductsOnboarding
```

Comment:
Looks like 1 extra tab in indentation?

Reply:
Good catch, fixed in https://github.com/woocommerce/woocommerce-ios/pull/12109/commits/b3ac496d57afdb00ea5a322df190b93cf88d0c1e